ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Initiator.java

```diff
@@ -331,6 +339,12 @@ private boolean foundCurrentOrFailedCompactions(ShowCompactResponse compactions,
     }
     return false;
   }
+  
+  private void updateDeltaFilesMetrics(AcidDirectory directory, String dbName, String tableName, String partName) {
```

Comment:
i think, this method could be extracted to a common place to be used by Cleaner and Initiator, like MetaStoreCompactorThread